simplesat/sat/policy.py

```diff
@@ -190,7 +190,7 @@ def get_next_package_id(self, assignments, clauses):
                     assignments
                 )
 
-        assert assignments[candidate_id] is None, \
+        assert assignments.value(candidate_id) is None, \
```

Comment:
Hmmm. Yes you're right. I had the relationship between ids and literals backwards. The keys in `AssignmentSet` are ids and thus always positive. `value` is used for looking up *literals*. Here we're looking up an id and should use `get`.